internal/engine/experiment/riseupvpn/riseupvpn.go

```diff
@@ -289,6 +299,8 @@ func generateMultiInputs(gateways []GatewayV3, transportType string) []urlgetter
 func parseGateways(testKeys *TestKeys) []GatewayV3 {
 	for _, requestEntry := range testKeys.Requests {
 		if requestEntry.Request.URL == eipServiceURL && requestEntry.Failure == nil {
+			// TODO(bassosimone,cyberta): is it reasonable that we discard
+			// the error when the JSON we fetched cannot be parsed?
```

Comment:
@cyBerta what do you think?

Reply:
From my understanding in that case the provider would be misconfigured or the API changed in an illegitimate way. I don't think that this case would show a blocking attempt. Raising the error would lead to false positives!?

Reply:
:thinking:

Reply:
Maybe we can introduce an error that clearly flags the backend as misconfigured, then? I suspect that not filling the fields in the data structure could lead to a confusing measurement result.

Reply:
It's a good idea. I think introducing a new error affects the spec, too. I would like to do that improvement as a separate PR

Reply:
Sure! I have created an issue tracking future improvements here: https://github.com/ooni/probe/issues/1432